pkg/athenz/util.go

```diff
@@ -20,3 +20,9 @@ func NamespaceToDomain(ns string) (domain string) {
 	dotted := strings.Replace(ns, "-", ".", -1)
 	return strings.Replace(dotted, "..", "-", -1)
 }
+
+func DomainToNamespaceForCloud(domain string) (namespace string) {
+	substring := strings.Split(domain, ".")
+	namespace = substring[len(substring)-1]
```

Comment:
Are we sure this is always true? Is this a presumption that everyone should follow this naming convention to make it work correctly?

Reply:
Thank you, for system namespaces mapped to Domain, I have used a different approach, please take a look again.